rustic-rustfmt.el

```diff
@@ -8,7 +8,9 @@
 
 (require 'rustic-cargo)
 
-(declare-function project-root "project")
+(if (version<= emacs-version "28.0")
+    (declare-function project-roots "project")
+  (declare-function project-root "project"))
```

Comment:
Do you think it makes sense to write a simple test that only checks if this is working. This change have been a little annoying #256

Reply:
Not sure how I would test this. Do you have any ideas?

Reply:
No, never mind. Maybe we add something later.